src/content/api/merchants.mdx

```diff
@@ -183,6 +196,10 @@ which define the payment methods available for a Payment Request.
       Merchant [Settlement Config](#settlement-config-model).
     </Property>
 
+    <Property name="refundConfig" type="object">
```

Comment:
Should we add the experimental tag as this will likely be a model that changes over time?